### PR TITLE
Improve initial instruction when opening the webpage

### LINKED ISSUE #1112: Improve initial instruction on web.threema.ch
When navigating to web.threema.ch, the instruction how to proceed is very short.

Currently the message is:
`Scan this QR code with your Threema app`
But it is not obvious, **where** in the app you do scanning.

I suggest something like:
`Scan this QR code with your Threema app.
To do this, start the app and tap on menu (icon "...") (Android) or Settings (icon "gear wheel") on iOS, then select "Threema Web" and follow the instructions there.`

This is a small change requiring only adaption of (translated) text.
The translation label is PLEASE_SCAN in
https://github.com/threema-ch/threema-web/blob/1656f800cd9db3bda8ced7991607deba1ddab1a4/public/i18n/en.json#L8
Maybe this is adapted with this translation label, maybe a 2nd label introduced (like PLEASE_SCAN_INSTR).

### PR DESCRIPTION
**Pull request by SimonHeimberg**
Fixes #1112

Having icons for "menu" and "Settings" would be nice. But this is a good step.

### CODE REVIEW

File: src/partials/welcome.html [hunk 1]
```diff
@@ -6,7 +6,9 @@
                 <p><strong translate>common.NOTE</strong> <span translate>welcome.SAFARI</span></p>
             </div>
 
-            <p class="instructions" translate>welcome.PLEASE_SCAN</p>
+            <p class="instructions" translate-attr="{'title:': 'welcome.PLEASE_SCAN_INSTR'}">
```
Comment (threema-danilo) [inline src/partials/welcome.html:9]: Stray colon in the title attribute name.

**Review by threema-danilo — CHANGES_REQUESTED** [review]
This doesn't actually work yet, did you test it? Besides the stray colon sign in the attribute name, the information icon is very large, not centered, and a bit distracting.

screenshot-20220425-102006

Maybe something like this would be better? (Font size is `.8em`)

**Comment by SimonHeimberg** [comment]
> This doesn't actually work yet, did you test it? Besides the stray colon sign in the attribute name, the information icon is very large, not centered, and a bit distracting.
> 

Sorry, no, I did not test it. I do not have the setup to do it. I have forgotten to mention this. The only thing I tested was to put the icon in the code (in the debug tools of the browser).

> 
> Maybe something like this would be better? (Font size is `.8em`)
> 
> screenshot-20220425-102754

I like your suggestion. It is short and clear. Will you implement it?

**Comment by threema-danilo** [comment]
Replaced by #1126.

**Pull request closed without merging**